HW-07/qw.py

```diff
@@ -0,0 +1,30 @@
+while True:
+    contacts = input('Enter contact: ')
+    con = contacts.split()
+
+    new_dic = {
```

Comment:
It is better to move this dictionary outside the loop since you override it with every iteration